CMakeLists.txt

```diff
@@ -128,6 +130,22 @@ if(LLVM_ENABLE_ZLIB)
   find_package(ZLIB)
 endif()
 
+#-------------------------------------------------------------------------------
+# Performance configuration
+#-------------------------------------------------------------------------------
+
+include(CheckCXXCompilerFlag)
+include(CheckLinkerFlag)
+add_link_options("$<$<BOOL:${STABLEHLO_ENABLE_LLD}>:-fuse-ld=lld>")
+if(STABLEHLO_ENABLE_SPLIT_DWARF)
+    check_cxx_compiler_flag(-gsplit-dwarf STABLEHLO_SUPPORTS_SPLIT_DWARF)
```

Comment:
Consider producing some kind of warning if split dwarf is not supported? Could help with debugging.